drivers/syslog/vsyslog.c

```diff
@@ -67,6 +67,7 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 {
   struct lib_syslogstream_s stream;
   int ret;
+  struct tcb_s *tcb;
```

Comment:
```suggestion
#if CONFIG_TASK_NAME_SIZE > 0 && defined(CONFIG_SYSLOG_PROCESS_NAME)
  struct tcb_s *tcb;
#endif
```
and remove line 221-222, since time/tm variable work this way.

Reply:
done